Library/Formula/sickrage.rb

```diff
@@ -0,0 +1,67 @@
+require "formula"
+
+class Sickrage < Formula
+  homepage "http://www.sickrage.tv/"
+  head "https://github.com/SiCKRAGETV/SickRage.git"
+  url "https://github.com/SiCKRAGETV/SickRage/archive/v4.0.9.tar.gz"
+  sha1 "77cbf0b7e5acbc7eaca4211c923d883db383fd71"
+
+  depends_on :python
+
+  resource "Cheetah" do
+    url "https://pypi.python.org/packages/source/C/Cheetah/Cheetah-2.4.4.tar.gz"
+    sha1 "c218f5d8bc97b39497680f6be9b7bd093f696e89"
+  end
+
+  def install
+    # install Sickrage contents into `libexec`
+    libexec.install Dir["*"]
+
+    # install Cheetah into `libexec`
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    resource("Cheetah").stage do
+      system "python", *Language::Python.setup_install_args(libexec)
+    end
+
+    # create SickRage executible
```

Comment:
Can remove all the comments in `install` as they are fairly self-explanatory, thanks.